clarify cwnd growth in convex region (#76)
NTAP/rfc8312bis: draft-ietf-tcpm-rfc8312bis.md

highly asynchronous, some amount of perturbation is always possible
without causing a major change in available bandwidth.

> Unless it is not overridden by AIMD window increase, CUBIC is very

Reviewer: ```suggestion
Unless it is not overridden by the AIMD window increase, CUBIC is very
```

> careful in this region. The convex profile aims that
> the window increases very slowly at the beginning when the *cwnd* is

Reviewer: ```suggestion
careful in this region. The convex profile aims to
increase the window very slowly at the beginning when *cwnd* is
```

> around *W<sub>max</sub>* and then gradually increases its increase rate.

Reviewer: ```suggestion
around *W<sub>max</sub>* and then gradually increases its rate of increase.
```
